msaf/utils.py

```diff
@@ -147,8 +147,15 @@ def sonify_clicks(audio, clicks, out_file, fs, offset=0):
     out_audio[:len(audio)] = audio
     out_audio[:len(audio_clicks)] += audio_clicks
 
+    # Peak normalize the mix
+    out_audio /= np.abs(out_audio).max()
+
+    # Convert audio to 16-bit signed integer
+    amplitude = np.iinfo(np.int16).max
+    data = (amplitude * out_audio).astype(np.int16)
```

Comment:
why `np.int16` and not `np.float16`?

Reply:
Because 32-bit audio cannot be stored at half-precision float without introducing quantization artifacts. Signed integer is necessary to retain audio fidelity.